package.json.in

```diff
@@ -52,12 +52,13 @@
         "@opam-alpha/merlin-extend": "^ 0.3.0",
         "@opam-alpha/menhir": ">= 20160303.0.0",
         "@opam-alpha/ocaml": "4.2.3",
+        "@opam-alpha/ocaml-migrate-parsetree": "*",
```

Comment:
Note to @andreypopp: I added `ocaml-migrate-parsetree` (brand new opam package) to our `opam-alpha` for `esy`, but not for `npm`. So when Fred lands this, maybe we should make Reason `esy` only.